packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -409,3 +418,35 @@ function exitWithNonZeroCodeIfSomeBuildsFailed(maybeBuilds: (BuildFragment | nul
     process.exit(1);
   }
 }
+
+async function downloadAndRunAsync(build: BuildFragment): Promise<void> {
+  assert(build.artifacts?.applicationArchiveUrl);
+  const buildPath = await downloadAndMaybeExtractAppAsync(
+    build.artifacts.applicationArchiveUrl,
+    build.platform
+  );
+  await runAsync(buildPath, build.platform);
+}
+
+async function maybeDownloadAndRunSimulatorBuildsAsync(
+  builds: MaybeBuildFragment[],
+  flags: BuildFlags
+): Promise<void> {
+  const simBuilds = builds.filter(truthy).filter(isRunnableOnSimulatorOrEmulator);
+
+  if (simBuilds.length > 0 && !flags.autoSubmit && !flags.nonInteractive) {
+    for (const simBuild of simBuilds) {
+      if (simBuild.platform === AppPlatform.Android || process.platform === 'darwin') {
+        Log.newLine();
+        const confirm = await confirmAsync({
+          message: `Would you like to run the ${
```

Comment:
what do you think about removing "Would you like to" here?

"Would you like to run the Android build on an emulator?"

->

"Install and run the Android build on an emulator?"

I think this is a bit more concise and to the point